lib/toTex.js

```diff
@@ -0,0 +1,102 @@
+/**
+   * toTex - return a string representation of the nodes in LaTeX
+   */
+
+const isNeg = (node) => {
+    return node.type === 'Operation' && node.op === 'neg' && node.args.length > 1;
+};
+
+const isAdd = (node) => {
+    return node.type === 'Operation' && node.op === 'add' && node.args.length > 1;
+};
+
+const isMul = (node) => {
+    return node.type === 'Operation' && node.op === '*' && node.args.length > 1;
+};
+
+function toTexOperation(node, parent){
+    let result;
+
+    switch(node.op){
+    case 'add':
+        //e.g a + (-a) => a - a       
+        result = print(node.args[0], node);
+        for (let i = 1; i < node.args.length; i++){
+            const arg = node.args[i];
+            if (isNeg(arg) && arg.wasMinus){
+                result += ` - ${print(arg.args[0], node)}`; 
+            } else {
+                result += ` + ${print(arg, node)}`;
+            }
+        }
+        return parent ? `(${result})` : result;
+    case 'neg':
+        return `-${print(node.args[0], node)}`;
+    case 'pos':
+        return `+${print(node.args[0], node)}`;
+    case 'pn':
+        throw new Error(`we don't handle 'pn' operations yet`);
+    case 'np':
+        throw new Error(`we don't handle 'np' operations yet`);
+    case 'mul':
+        if (node.implicit) {
+            //e.g 2 x 
+            return node.args.map(arg => print(arg, node)).join(` `);
+        } else {
+            //e.g 2 * x
+            return node.args.map(arg => print(arg, node)).join(` \ast `); // ast for asterisk and \time for x
+        }
+    case 'div':
+        result = '';
+        result += '\frac';
+        //add parentheses when numerator or denominator has multiple terms
+        //e.g latex fractions: \frac{a}{b} => a/b
+        if (isAdd(node.args[0]) || isMul(node.args[0])){
+            result += `{(${print(node.args[0], node)})}`;
```

Comment:
the parens aren't necessary here b/c the fraction will indicate that the order of operations